src/server_manager/web_app/ui_components/app-root.ts

```diff
@@ -391,6 +395,11 @@ export class AppRoot extends polymerElementWithLocalize {
 
           <!-- Links section -->
           <paper-listbox>
+            <a class="documentation-link" 
+              href="https://www.reddit.com/r/outlinevpn/wiki/index/">
+                <span>[[localize('outline-documentation')]]</span>
```

Comment:
Should this be next to the field title instead?
I imagine something like:

    Management API URL (documentation)
    https://123.123.123.123:1000/fjsdkalfjfsdajfkljasdlkf/

Where `documentation` is a link to the Wiki.

Reply:
Moving to another PR sounds good. We can explore the new components perhaps: https://material-web.dev/components/text-field/